services/kuber/server/domain/utils/nodes/patch.go

```diff
@@ -69,18 +69,20 @@ func (p *Patcher) PatchLabels() error {
 	for _, np := range p.desiredNodepools {
 		l, err1 := nodes.GetAllLabels(np, nil)
```

Comment:
```suggestion
		allLabels, err1 := nodes.GetAllLabels(np, nil)
```
Just a suggestion to make the variable name more self-explanatory.